### ISSUE TITLE
add port to output file

### ISSUE THREAD

**Issue by SirAlidadi** [idealeer/xmap#16]
Hello, how can I add the port to the end of the scanned ports in the output?
Is there such a feature?

**Comment by idealeer** [comment]
yes. --output-fields option provides choices to select any output fields in the output.

**Comment by SirAlidadi** [comment]
./xmap -i enp35s0 -4 -R 0 -p 3388,8080 -O csv --probe-args=file:rang.txt -o iplist.txt -T9 --output-fields saddr,port

This command gives an error because the port field is wrong. What field should I enter for port?

**Comment by SirAlidadi** [comment]
![IMG_20230915_183205_463](https://github.com/idealeer/xmap/assets/82961992/9c0adc1f-cb2e-4fa3-9504-f873bbb9575b)

**Comment by idealeer** [comment]
please use `sport` or `dport`.
